contracts/ProtocolReserve/ProtocolShareReserve.sol

```diff
@@ -210,12 +213,61 @@ contract ProtocolShareReserve is
         _ensureMaxLoops(distributionTargets.length);
     }
 
+    /**
+     * @dev Remove destionation target if percentage is 0
+     * @param schema schema of the configuration
+     * @param destination destination address of the configuration
+     */
+    function removeDistributionConfig(Schema schema, address destination) external {
+        _checkAccessAllowed("removeDistributionConfig(Schema,address)");
```

Comment:
We would need to consider this new function in the VIP, grant Normal, Fast-track and Critical timelocks to execute it